spec/duration.html

```diff
@@ -511,15 +513,32 @@ <h1>Temporal.Duration.prototype.total ( _totalOf_ )</h1>
           1. Set _plainRelativeTo_ to ! CreateTemporalDate(_precalculatedPlainDateTime_.[[ISOYear]], _precalculatedPlainDateTime_.[[ISOMonth]], _precalculatedPlainDateTime_.[[ISODay]], _zonedRelativeTo_.[[Calendar]]).
         1. Let _calendarRec_ be ? CreateCalendarMethodsRecordFromRelativeTo(_plainRelativeTo_, _zonedRelativeTo_, « ~dateAdd~, ~dateUntil~ »).
         1. Let _unbalanceResult_ be ? UnbalanceDateDurationRelative(_duration_.[[Years]], _duration_.[[Months]], _duration_.[[Weeks]], _duration_.[[Days]], _unit_, _plainRelativeTo_, _calendarRec_).
+        1. Let _days_ be _unbalanceResult_.[[Days]].
         1. If _zonedRelativeTo_ is not *undefined*, then
           1. Let _intermediate_ be ? MoveRelativeZonedDateTime(_zonedRelativeTo_, _calendarRec_, _timeZoneRec_, _unbalanceResult_.[[Years]], _unbalanceResult_.[[Months]], _unbalanceResult_.[[Weeks]], 0, _precalculatedPlainDateTime_).
-          1. Let _balanceResult_ be ? BalancePossiblyInfiniteTimeDurationRelative(_unbalanceResult_.[[Days]], _duration_.[[Hours]], _duration_.[[Minutes]], _duration_.[[Seconds]], _duration_.[[Milliseconds]], _duration_.[[Microseconds]], _duration_.[[Nanoseconds]], _unit_, _intermediate_, _timeZoneRec_).
+          1. Let _norm_ be NormalizeTimeDuration(_duration_.[[Hours]], _duration_.[[Minutes]], _duration_.[[Seconds]], _duration_.[[Milliseconds]], _duration_.[[Microseconds]], _duration_.[[Nanoseconds]]).
+          1. Let _startInstant_ be ! CreateTemporalInstant(_intermediate_.[[Nanoseconds]]).
+          1. Let _startDateTime_ be *undefined*.
+          1. If _days_ &ne; 0, then
+            1. Set _startDateTime_ to ? GetPlainDateTimeFor(_timeZoneRec_, _startInstant_, *"iso8601"*).
+            1. Let _addResult_ be ? AddDaysToZonedDateTime(_startInstant_, _startDateTime_, _timeZoneRec_, *"iso8601"*, _days_).
+            1. Let _intermediateNs_ be _addResult_.[[EpochNanoseconds]].
+          1. Else,
+            1. Let _intermediateNs_ be _intermediate_.[[Nanoseconds]].
+          1. Let _endNs_ be AddInstant(_intermediateNs_, _norm_).
+          1. Set _norm_ to NormalizedTimeDurationFromEpochNanosecondsDifference(_endNs_, _intermediateNs_).
```

Comment:
```suggestion
          1. Set _norm_ to NormalizedTimeDurationFromEpochNanosecondsDifference(_endNs_, _intermediate_.[[Nanoseconds]]).
```

Also matches the spec polyfill.